### ISSUE TITLE
I have a question about scalability.

### ISSUE THREAD

**Issue by JiayunjieJYJ** [ZiyuanMa/DHC#1]
Hi, if I train a model in an environment containing 5 agents, can I test the model in a new environment containing 10 agents without any change of the code of the model?

**Comment by ZiyuanMa** [comment]
Yes, the number of agents in an environment does not matter, you can train and test with any number of agents.

**Comment by JiayunjieJYJ** [comment]
You use ray package to make each robot to parallelly interact with the common environment, right?

**Comment by JiayunjieJYJ** [comment]
Is the number of workers (class Actor) the same as the number of agents in one environment?

**Comment by ZiyuanMa** [comment]
No, each actor runs an independent environment, and each environment may contain a different number of agents.
